eng/pipelines/windows.yml

```diff
@@ -50,6 +50,12 @@ jobs:
               _architecture: x64
               _framework: uap
               _helixQueues: $(uapNetfxQueues)
+
+            arm_Debug:
+              _BuildConfig: Debug
+              _architecture: arm
+              _framework: netcoreapp
+              _helixQueues: $(windowsArmQueue)
```

Comment:
Plan is to remove these lines before merging since we don't have enough hardware to run this on every PR.